Gordon360/Models/ViewModels/RecIM/UpdateMatchTeamViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Gordon360.Models.CCT;
+using System;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class UpdateMatchTeamViewModel
+    {
+        public int TeamID { get; set; }
+        public int MatchID { get; set; }
+        public string? Status { get; set; }
+        public int? Score { get; set; }
+        public int? Sportsmanship { get; set; }
```

Comment:
`Sportsmanship` as a field name doesn't shout "INT" to me. Is there a more descriptive name for what you are trying to track with this field?